dependency_support/org_theopenroadproject_asap7/pdn_config.pdn

```diff
@@ -15,11 +15,11 @@ set ::ground_nets "VSS"
 ####################################
 # global connections
 ####################################
-add_global_connection -defer_connection -net {VDD} -inst_pattern {.*} -pin_pattern {^VDD$} -power
```

Comment:
is that also required because of the non-scaled variant? if not, can you add a line about this in the PR description?

Reply:
`-defer_connection` was removed in ORFS because this option is deprecated in newer `openroad`. However, the version used in `bazel_rules_hdl` still has it so it would be reasonable to keep that.